mmv1/third_party/terraform/tpgresource/import.go

```diff
@@ -21,25 +21,16 @@ import (
 func ParseImportId(idRegexes []string, d TerraformResourceData, config *transport_tpg.Config) error {
 	for _, idFormat := range idRegexes {
 		re, err := regexp.Compile(idFormat)
-
 		if err != nil {
 			log.Printf("[DEBUG] Could not compile %s.", idFormat)
 			return fmt.Errorf("Import is not supported. Invalid regex formats.")
 		}
-
-		if d.Id() == "" {
-			identity, err := d.Identity()
-			if err != nil {
-				return err
-			}
-			if err := identityImport(re, identity, idFormat, d); err != nil {
-				return err
-			}
-			err = setDefaultValues(idRegexes[0], identity, d, config)
-			if err != nil {
-				return err
-			}
-		} else if fieldValues := re.FindStringSubmatch(d.Id()); fieldValues != nil {
+		identity, err := d.Identity()
+		if err != nil {
+			return err
+		}
+		log.Printf("[DEBUG] Identity Schema %s", identity)
```

Comment:
This log is for development and can be removed.

The same for other development logs below.